### ISSUE TITLE
Hello! how to fix this?

### ISSUE THREAD

**Issue by f908** [elebumm/RedditVideoMakerBot#1167]
**Describe the bug**
Using Poland server backend.

██████╗ ███████╗██████╗ ██████╗ ██╗████████╗    ██╗   ██╗██╗██████╗ ███████╗ ██████╗     ███╗   ███╗ █████╗ ██╗  ██╗███████╗██████╗
██╔══██╗██╔════╝██╔══██╗██╔══██╗██║╚══██╔══╝    ██║   ██║██║██╔══██╗██╔════╝██╔═══██╗    ████╗ ████║██╔══██╗██║ ██╔╝██╔════╝██╔══██╗
██████╔╝█████╗  ██║  ██║██║  ██║██║   ██║       ██║   ██║██║██║  ██║█████╗  ██║   ██║    ██╔████╔██║███████║█████╔╝ █████╗  ██████╔╝
██╔══██╗██╔══╝  ██║  ██║██║  ██║██║   ██║       ╚██╗ ██╔╝██║██║  ██║██╔══╝  ██║   ██║    ██║╚██╔╝██║██╔══██║██╔═██╗ ██╔══╝  ██╔══██╗
██║  ██║███████╗██████╔╝██████╔╝██║   ██║        ╚████╔╝ ██║██████╔╝███████╗╚██████╔╝    ██║ ╚═╝ ██║██║  ██║██║  ██╗███████╗██║  ██║
╚═╝  ╚═╝╚══════╝╚═════╝ ╚═════╝ ╚═╝   ╚═╝         ╚═══╝  ╚═╝╚═════╝ ╚══════╝ ╚═════╝     ╚═╝     ╚═╝╚═╝  ╚═╝╚═╝  ╚═╝╚══════╝╚═╝  ╚═╝

     Thanks for using this tool! Feel free to contribute to this project on GitHub! If you have any questions, feel free to reach out to me on
                       Twitter or submit a GitHub issue. You can find solutions to many common problems in the Documentation

╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ You are using an older version (2.4.1) of the bot. Download the newest version (2.4.2) from                                                      │
│ https://github.com/elebumm/RedditVideoMakerBot/releases/latest                                                                                   │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
###############################
#                             #
# Checking TOML configuration #
#                             #
###############################
If you see any prompts, that means that you have unset/incorrectly set variables, please input the correct values.
╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ on the 1st iteration of 3                                                                                                                        │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
Logging into Reddit.
╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ Getting subreddit threads...                                                                                                                     │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
Using subreddit: r/AskReddit from TOML config
NSFW Post Detected. Skipping...
Video will be: what is something you silently judge people for? 👍
Thread has 17282 upvotes
Thread has a upvote ratio of 92.0%
Thread has 13493 comments
Received subreddit threads Successfully.
Thread ID is wp3p92
╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ Saving Text to MP3 files...                                                                                                                      │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
Translating Text...
Translating Text...
Saving... ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   Translating Text...
Saving... ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   Translating Text...
Saving... ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   Translating Text...
Saving... ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   Translating Text...
Saving... ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 0.1 it/s  Translating Text...
Saving... ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 0.1 it/s  Translating Text...
Saving... ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 0.2 it/s
Saved Text to MP3 files successfully.
╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ Downloading screenshots of reddit posts...                                                                                                       │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
Launching Headless Browser...
Translating post...
Downloading screenshots... ━╸━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   4% 0:10:08^R
Downloading screenshots... ━━╺━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   6% 0:10:26
Screenshots downloaded Successfully.
╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ We need to download the backgrounds videos. they are fairly large but it's only done once. 😎                                                    │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
Downloading the backgrounds videos... please be patient 🙏
Downloading rocket_league.mp4 from https://www.youtube.com/watch?v=2X9QGY__0II
Traceback (most recent call last):█████████████                                      | 53.0%
  File "/home/ubuntu/RedditVideoMakerBot/main.py", line 83, in <module>
    run_many(config["settings"]["times_to_run"])
  File "/home/ubuntu/RedditVideoMakerBot/main.py", line 62, in run_many
    main()
  File "/home/ubuntu/RedditVideoMakerBot/main.py", line 52, in main
    download_background(bg_config)
  File "/home/ubuntu/RedditVideoMakerBot/video_creation/background.py", line 60, in download_background
    YouTube(uri, on_progress_callback=on_progress).streams.filter(res="1080p").first().download(
  File "/home/ubuntu/.local/lib/python3.10/site-packages/pytube/streams.py", line 252, in download
    for chunk in request.stream(
  File "/home/ubuntu/.local/lib/python3.10/site-packages/pytube/request.py", line 185, in stream
    chunk = response.read()
  File "/usr/lib/python3.10/http/client.py", line 481, in read
    s = self._safe_read(self.length)
  File "/usr/lib/python3.10/http/client.py", line 632, in _safe_read
    raise IncompleteRead(data, amt-len(data))
http.client.IncompleteRead: IncompleteRead(8608640 bytes read, 828544 more expected)

**To Reproduce**
I don't know.

**Expected behavior**
For everything to go well.

**Screenshots**
[Click here](https://chillovo.ml/dl/redditbot.png)

**System (please complete the following information):**

- Python Version: Python 3.10.4
- OS: Ubuntu 22.04 LTS
- App version / Branch: master

**Additional context**

**Comment by f908** [comment]
TL;DR: Downloading Rocket League Failed.

**Comment by StopmotionSimonYT** [comment]
Hi, if it's only with the Rocket League background, you can remove it! Go to utils/CONSTANTS.py and remove the background.

**Comment by f908** [comment]
> Hi, if it's only with the Rocket League background, you can remove it! Go to utils/CONSTANTS.py and remove the background.

oh ok, i will try later

**Comment by f908** [comment]
Nah i'm giving up, it's still broken.
